doc/annotations.md

```diff
@@ -0,0 +1,94 @@
+# Sources and contentType
+All annotations associated with Containerized Data Importer (CDI) have a prefix of: cdi.kubevirt.io. This denotes that the annotations are meant to be consumed by the CDI controller.
+
+## Source
+Source describes the type of data source CDI will be collecting the data from. Based on the value of source, additional annotations may be required to successfully import the data. The full annotation for source is: cdi.kubevirt.io/storage.import.source. The following values are currently available:
+* http
```

Comment:
maybe worth referring to Source constants in util file, so we won't have to update this file when adding a new one (or we prefer to keep the documentation neutral from the code?)

Reply:
We will have to update this file when we add new sources, we don't want users to go and dive into sources to figure out which sources are valid.